src/IO/S3/copyS3File.cpp

```diff
@@ -184,7 +188,7 @@ namespace
                     LOG_INFO(log, "Multipart upload failed with NO_SUCH_KEY error for Bucket: {}, Key: {}, Upload_id: {}, Parts: {}, will retry", dest_bucket, dest_key, multipart_upload_id, part_tags.size());
                     continue; /// will retry
                 }
-
+                ProfileEvents::increment(ProfileEvents::WriteBufferFromS3RequestsErrors, 1);
```

Comment:
same in `createMultipartUpload` before exception?

Reply:
Yes. Added. Thanks!